runtime/doc/msgpack_rpc.txt

```diff
@@ -242,4 +242,170 @@ Even for statically compiled clients it is good practice to avoid hardcoding
 the type codes, because a client may be built against one Nvim version but
 connect to another with different type codes.
 
+==============================================================================
+6. Buffer Updates				      *buffer-updates* *rpc-buffer-updates*
+
+A dedicated API has been created to allow co-processes to be notified  when a
+buffer is changed in any way. It is difficult and error-prone to try and do
+this with autocommands such as |TextChanged|.
+
+                                                              *buffer-updates-api*
+BufferUpdates Functions~
+
+nvim_buf_updates_start({send_buffer})                   *nvim_buf_updates_start()*
```

Comment:
Function docs with argument lists will be auto-generated from the doxygen docs, so these shouldn't be duplicated here. See `:help api-buffer` for example of the results. (Which means it's also fine for the docs inside `api/*.c` to use helptags, so the reference to the events can be moved there)

Reply:
I'm being a bit dense, sorry. I've looked at `:h api-buffer`, and it looks fine. But that document does not contain `nvim_buf_updates_start`, do I need to do something special to get it in there? I'll scratch those function definitions, but please let me know what I need to do for the api docs to work.

Reply:
They will be generated before release at latest, so don't worry.

Reply:
But `scripts/gen_api_vimdoc.py` if you're curious

Reply:
Ahh now that's comfortable :)